src/services/pool.ts

```diff
@@ -0,0 +1,112 @@
+import { Db, Filter, FindOptions, MongoClient, WithId } from 'mongodb';
+import { createMessage, initiateLogger } from './logger';
+import { assertTrailingSlash } from '../utils';
+
+/** BEGIN typing for DB */
+
+type EnvKeyedObject = {
+  prd: string;
+  preprd: string;
```

Comment:
Should this be `stg`? 

But also, wondering if we ever need anything other than the `dotcomstg` and `dotcomprd` envs given the data needed from `pool`.

Reply:
good catch!